packages/customize-widgets/src/components/customize-widgets/index.js

```diff
@@ -42,13 +43,15 @@ export default function CustomizeWidgets( {
 	const activeSidebar =
 		activeSidebarControl &&
 		createPortal(
-			<SidebarBlockEditor
-				key={ activeSidebarControl.id }
-				blockEditorSettings={ blockEditorSettings }
-				sidebar={ activeSidebarControl.sidebarAdapter }
-				inserter={ activeSidebarControl.inserter }
-				inspector={ activeSidebarControl.inspector }
-			/>,
+			<ErrorBoundary>
```

Comment:
I wonder if we should add this to the root of the editor, in hope of being able to correctly reset all the states. Not sure if it would work though.

Reply:
I gave it a try during development, but yeah, doesn't quite work. I think we'll need something a little different to handle the portals properly in the customizer editor.

Reply:
I don't think it's a problem with the portals, but rather that we(I) don't properly clear/reset the states after unmounted 😅. It didn't occur to me that we would ever need to unmount the component. Probably just have to add some `useEffect` in the components.

Reply:
The error boundary UI also needs to render in the right place (which is probably in the portal).

7d607ff has my first attempt to solve the `activeSidebarControl` state problems, but it was a bit naive because even the context is lost when unmounting/remounting the app.

Moving the active sidebar control to store state so that it persists across umounting/remounting the whole application might be the easiest option.